Template/protocol_template.py

```diff
@@ -0,0 +1,207 @@
+from opentrons import protocol_api
+
+metadata = {
+    'protocolName': 'Protocol Title',
+    'author': 'AUTHOR NAME <authoremail@company.com>',
+    'source': 'Custom Protocol Request',
+    'apiLevel': '2.11'   # CHECK IF YOUR API LEVEL HERE IS UP TO DATE
+                         # IN SECTION 5.2 OF THE APIV2 "VERSIONING"
+}
+
+
+def run(ctx: protocol_api.ProtocolContext):
+
+    [
+     custom_variable1,
+     custom_variable2
+    ] = get_values(  # noqa: F821 (<--- DO NOT REMOVE!)
+        "custom_variable1",
+        "custom_variable2")
+
+    if not 1 <= custom_variable1 <= 12:
+        raise Exception("Enter a value between 1-12")
+
+    # define all custom variables above here with descriptions:
+
+    # number of samples
+    custom_variable1 = custom_variable1
+
+    # "True" for park tips, "False" for discard tips
+    custom_variable2 = custom_variable2
+
+    # load modules
+
+    '''
+
+    Add your modules here with:
+
+    module_name = ctx.load_module('{module_loadname}', '{slot number}')
+
+    Note: if you are loading a thermocycler, you do not need to specify
+    a slot number - thermocyclers will always occupy slots 7, 8, 10, and 11.
+
+    For all other modules, you can load them on slots 1, 3, 4, 6, 7, 9, 10.
+
+    '''
+
+    # load labware
+
+    '''
+
+    Add your labware here with:
+
+    labware_name = ctx.load_labware('{loadname}', '{slot number}')
+
+    If loading labware on a module, load with:
+
+    labware_name = module_name.load_labware('{loadname}')
+    where module_name is defined above.
+
+    '''
+
+    # load tipracks
```

Comment:
I noticed that in some protocols the pipettes are loaded first, and then the tipracks are heuristically determined based on what volumes the pipettes can handle. Do we want that in our template?

Reply:
We could load the pipettes first, and then the writer could make that call themselves.